Sources/BothamPullToRefreshHandler.swift

```diff
@@ -36,8 +39,14 @@ public class BothamPullToRefreshHandler: NSObject {
         self.refreshControl.endRefreshing()
     }
 
-    public func beginRefreshing(scrollView: UIScrollView) {
-        scrollView.setContentOffset(CGPoint(x: 0, y: -refreshControl.frame.size.height), animated: true)
-        self.refreshControl.beginRefreshing()
+    public func beginRefreshing() {
+        if !refreshControl.refreshing {
```

Comment:
Probably the best idea here is to match the refreshControl property, "refreshing", instead of a new method, isRefreshing. So more or less:
public var refreshing: Bool { 
     return refreshControl.refreshing
}